### PR TITLE
Adds zh-CN translation for some sections in README

### PR DESCRIPTION
**Pull request by inkuxuan**

### CODE REVIEW

File: docs/readmes/src/cn/texts.txt [hunk 1]
```diff
@@ -262,55 +262,101 @@ OyasumiVR的开发者和此项目的所有贡献者对OyasumiVR造成的任何
 OyasumiVR在[MIT](https://github.com/Raphiiko/OyasumiVR/blob/develop/LICENSE.md)许可证下发布。你可以在许可证许可范围内自由使用、修改和分发它。
 
 {{SUPPORT_OYASUMIVR_HEADER}}
-
+支援Oyasumi VR
```
Comment (flower-elf) [inline docs/readmes/src/cn/texts.txt:265]: 在简体中文中一般使用“支持”而不是“支援”，在此处也可以使用“赞助”一类的词语

File: docs/readmes/src/cn/texts.txt [hunk 2]
```diff
@@ -319,7 +365,7 @@ OyasumiVR在[MIT](https://github.com/Raphiiko/OyasumiVR/blob/develop/LICENSE.md)
 
 
 {{SUPPORTED_HARDWARE_BRIGHTNESS_AUTOMATIONS_BODY}}
-
+控制头显亮度目前只支持 Valve Index 和 Bigscreen Beyond。
```
Comment (flower-elf) [inline docs/readmes/src/cn/texts.txt:368]:
```suggestion
头显亮度控制功能目前只支持 Valve Index 和 Bigscreen Beyond。
```

**Review by flower-elf — CHANGES_REQUESTED** [review]

File: docs/readmes/src/cn/texts.txt (hunk 1 above)
Comment (inkuxuan) [reply docs/readmes/src/cn/texts.txt]: Rewording suggestion committed by 8fbcc1999125a016227f17a742c41ce037280b31

File: docs/readmes/src/cn/texts.txt (hunk 2 above)
Comment (inkuxuan) [reply docs/readmes/src/cn/texts.txt]: Rewording suggestion committed by 8fbcc1999125a016227f17a742c41ce037280b31

**Review by flower-elf — APPROVED** [review]

**Comment by Raphiiko** [comment]
Thanks for the updates!

**Pull request merged**